lib/rentlinx.rb

```diff
@@ -1,7 +1,11 @@
 require 'rentlinx/client'
+require 'rentlinx/models/property'
 
 # The Rentlinx module provides the means to interact with the Rentlinx API.
 module Rentlinx
+  class ProgrammerError < StandardError
```

Comment:
What does this error represent? Is there a better name for it?

Also, should we avoid adding too much to the `lib/rentlinx.rb` file? I envision that at some point there should be a base Rentlinx error class (either `Rentlinx::Error`, or `Rentlinx::RentlinxError`) that all other Rentlinx related error classes inherit from. Once that's added, it's encouraged to be very liberal with the number of error classes added.

That gives users of the library the flexibility to handle errors very granularly (`catch Rentlinx::Error`), or very precisely (`catch RentLinx::TooManyAmenenities`).